r/R/dataset-scan.R

```diff
@@ -183,6 +191,10 @@ ScannerBuilder <- R6Class("ScannerBuilder", inherit = ArrowObject,
       dataset___ScannerBuilder__UseThreads(self, threads)
       self
     },
+    UseAsync = function(use_async = FALSE) {
```

Comment:
Did you mean to default to TRUE?

Reply:
(Otherwise the parameter above doesn't work.)

Reply:
Oops, good catch